flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/observer/deployment/AbstractDeploymentObserver.java

```diff
@@ -98,6 +98,10 @@ public void observe(FlinkDeployment flinkApp, Context context) {
             }
         }
 
+        if (!ReconciliationUtils.isJobRunning(flinkApp.getStatus())) {
```

Comment:
We might also need to create an event when we canceling the savepoint operation. At least, we need to add a log when doing a concrete reset(when the `triggerId` is not empty).

Reply:
Added, PTAL